src/commands/trialApp/importTrialApp.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { commands, ProgressLocation, window } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { TrialAppLoginSession } from '../../constants';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../localize';
+
+export async function importTrialApp(_context: IActionContext, loginSession?: string): Promise<void> {
```

Comment:
If loginSession is required, I don't think you should make it optional parameter.  You should have the logic to make sure that it is a string when you call it anyway.

Reply:
No longer optional 👍